pyproject.toml

```diff
@@ -21,18 +21,7 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
 # NOTE: The below is managed by ./tools/update-optional-dependencies.py
-dependencies = [
-    "bentoml[grpc,io]>=1.0.22",
-    "transformers[torch,tokenizers,accelerate]>=4.29.0",
-    "optimum",
-    "attrs>=23.1.0",
-    "cattrs>=23.1.0",
-    "orjson",
-    "inflection",
-    "tabulate[widechars]>=0.9.0",
-    "httpx",
-    "typing_extensions",
-]
```

Comment:
Can you avoid this change, because it will be managed by `./tools/update-optional-dependencies.py`